forms-flow-bpm/src/main/resources/processes/eao-workflows/assessment-process.bpmn

```diff
@@ -501,12 +521,252 @@
         <camunda:executionListener class="org.camunda.bpm.extension.hooks.listeners.SyncFormDataPipelineListener" event="take" />
       </bpmn:extensionElements>
     </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_0zci9bl" sourceRef="Gateway_0kyl34x" targetRef="Activity_1cxdn4p">
+      <bpmn:extensionElements>
+        <camunda:executionListener event="take">
+          <camunda:script scriptFormat="javascript">execution.setVariable('applicationStatus', "Accept IPD &amp; Engagement Plan");</camunda:script>
```

Comment:
Also, instead of hard coding the event name, can you get it from the variable itself. 
`execution.getVariable('completedEvent')`